arkouda/groupbyclass.py

```diff
@@ -210,7 +210,10 @@ def aggregate(self, values : pdarray, operator : str, skipna : bool=True) \
         ValueError
             Raised if the key array size does not match the values size or
             if the operator is not in the GroupBy.Reductions array
-
+ 
+        Examples
+        --------
+        
```

Comment:
There is an example in the pydoc, maybe it should be included here also

Reply:
@mhmerrill cool, will add it

Reply:
@mhmerrill added examples extensively across groupbyclass and added dtype checking to prevent roundtrips for unsupported dtypes. Unless you or @reuster986 have any further changes you'd like to see, I recommend closing out work on this PR.